src/ScheduleService.Models.Db/DbTask.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Db;
+
+public class DbTask
+{
+  public const string TableName = "Tasks";
+
+  public Guid Id { get; set; }
+  public string Name { get; set; }
+  public string Description { get; set; }
+  public int Type { get; set; } //enum
+  public Guid GroupId { get; set; }
+  public Guid WorkspaceId { get; set; }
```

Comment:
@SolovyovDmitry is it normal that tables do not contain fields(CreatedBy, ModifiedBy ...)?

Reply:
Same about order

Reply:
As @VLiamin said, you need to add CreatedBy,CreatedAtUtc columns for all entities; ModifiedBy, ModifiedAtUtc for entities that can be edited; IsActive for entities that can be archived(soft deleted).!!! Except intermediate many-to-many tables

Reply:
Do i need to just add it as other properties or with additional logic? And there are only five of them?

Reply:
+